plugins/package-managers/nuget/src/main/kotlin/utils/NuGetUtils.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.nuget.utils
+
+import org.ossreviewtoolkit.model.Identifier
+
+fun getIdentifierWithNamespace(type: String, name: String, version: String): Identifier {
```

Comment:
Hm, I believe I'm missing something, maybe you can help me finding out from where (besides the test) this (new) function is called.

Reply:
See a few comments [above](https://github.com/oss-review-toolkit/ort/pull/6046#discussion_r1314748990).

Reply:
Sorry, not following you.

Reply:
Did you follow the link? The code needs to be public in order to be able to call it from the code in `ort-config` that generates the NuGet curations. (And then these curations need to be regenerated, obviously.)

Reply:
> Did you follow the link? 

Yes, but hadn't noticed it was a different repo.

Reply:
> Hm, I believe I'm missing something, maybe you can help me finding out from where (besides the test) this (new) function is called.

Ah, now I realize what you were confused by: Due to a wrong conflict resolution while rebasing, indeed the actual call to `getIdentifierWithNamespace()` got lost. I've fixed that now (plus doing some more other fixes).